app/javascript/menu/menu-collapse.jsx

```diff
@@ -1,26 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { ChevronLeft20, ChevronRight20 } from '@carbon/icons-react';
-import { SideNavItem } from 'carbon-components-react/es/components/UIShell';
+import { SideNavItems, SideNavItem } from 'carbon-components-react/es/components/UIShell';
 
 const MenuCollapse = ({
   expanded, toggle, onFocus, open,
 }) => (
-  <SideNavItem className="menu-collapse">
-    <div
-      role="button"
-      tabIndex="0"
-      className="menu-collapse-button"
-      onClick={toggle}
-      onKeyPress={toggle}
-      onFocus={onFocus}
-      aria-expanded={expanded}
-      aria-controls="main-menu-primary"
-      aria-haspopup="true"
-    >
-      {(expanded && !open) ? <ChevronLeft20 /> : <ChevronRight20 />}
-    </div>
-  </SideNavItem>
+  <div className="menu-collapse">
+    <SideNavItems className="menu-collapse-list">
```

Comment:
Does this introduce a UI change?  Can you show before/after screenshots if so?

Reply:
Before, the `<li>` did not have the `<ul>` as its parent element. 
After introducing the `SideNavItems`, the `<ul>` tag is added to `<li>`

Reply:
there is no notable ui change. its the button at the bottom navigation
![image](https://user-images.githubusercontent.com/87487049/210721339-cde7f0fa-7811-4516-ae95-5ea7963d038c.png)